eng/common/pipelines/templates/jobs/eng-common-workflow-enforcer.yml

```diff
@@ -0,0 +1,27 @@
+trigger: none
+
+pr:
```

Comment:
This looks like you are planning to run this as its own job. I'm not sure I'd go that far. I'd suggest instead we add it as a template step and include it in the correct pipelines in the language repo. Usually the core or the template pipeline but really any pipeline that will trigger based on eng/common changes.

Reply:
My idea was to use the pipeline pr trigger functionality, so the job should not even run if there is not change to eng/common

Reply:
I'm not convinced that having a dedicated pipeline is worth it for this check.